Makefile

```
@@ -1166,7 +1161,7 @@ release-notes-tool:
 
 .PHONY: release-notes
 release-notes: release-notes-tool
-	./bin/notes --release $(RELEASE_TAG) > CHANGELOG/$(RELEASE_TAG).md
+	./bin/notes --release $(RELEASE_TAG) --previous-release-version "$(PREVIOUS_VERSION_TAG)" > CHANGELOG/$(RELEASE_TAG).md
```

```suggestion
	./bin/notes --release $(RELEASE_TAG) --previous-release-version "$(PREVIOUS_RELEASE_TAG)" > CHANGELOG/$(RELEASE_TAG).md
```

For consistency. Please change this for all occurences of PREVIOUS_VERSION_TAG. Feel free to ping me afterwards and I'll merge the PR